packages/workflow/src/utils.ts

```diff
@@ -30,3 +30,20 @@ export const deepCopy = <T>(source: T): T => {
 	return clone;
 };
 // eslint-enable
+
+export const parseJson = (
+	jsonString: string,
+	options?: { errorMessage?: string; fallbackValue?: any },
+) => {
+	try {
+		return JSON.parse(jsonString);
```

Comment:
can we rename this to `parseJSON` please, and also add a generic parameter here to avoid having to cast the result with `as`. Then in a lot of places, `<T>` can simply be inferred, and no explicit casting is needed.
```suggestion
export const parseJSON = <T>(
	jsonString: string,
	options?: { errorMessage?: string; fallbackValue?: any },
) => {
	try {
		return JSON.parse(jsonString) as T;
```